### ISSUE TITLE
About KL Divergence loss

### ISSUE THREAD

**Issue by ylzz1997** [jaywalnut310/vits#155]
About KL Divergence loss in [losses.py](https://github.com/jaywalnut310/vits/blob/main/losses.py) 

```
def kl_loss(z_p, logs_q, m_p, logs_p, z_mask):
  z_p = z_p.float()
  logs_q = logs_q.float()
  m_p = m_p.float()
  logs_p = logs_p.float()
  z_mask = z_mask.float()

  kl = logs_p - logs_q - 0.5
  kl += 0.5 * ((z_p - m_p)**2) * torch.exp(-2. * logs_p)
  kl = torch.sum(kl * z_mask)
  l = kl / torch.sum(z_mask)
  return l
```

KL Loss:
```
kl = logs_p - logs_q - 0.5
kl += 0.5 * ((z_p - m_p)**2) * torch.exp(-2. * logs_p)
```

The formula for calculating Gaussian divergence is:
![{{2J%WV3 TT`@{XQE$81%QN](https://github.com/jaywalnut310/vits/assets/28924547/7022e54d-f030-4af9-938c-ab5a41d261ac)

But the code is:
![~4XAV V4)Z56%Q(}50V5M7K](https://github.com/jaywalnut310/vits/assets/28924547/8e35223b-eabc-4e09-b30f-306a709a2569)

loss in code $σ_1^2$

May I ask why $σ_1^2$ is missing?

Question quoted in 

> https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/issues/469

**Comment by nshmyrev** [comment]
Same as https://github.com/jaywalnut310/vits/issues/6#issuecomment-861903556
